pkg/deploy/dev-workspace-config/dev_workspace_config.go

```diff
@@ -71,8 +71,10 @@ func (d *DevWorkspaceConfigReconciler) Finalize(ctx *chetypes.DeployContext) boo
 	return true
 }
 
-func updateOperatorConfig(storage chev2.WorkspaceStorage, operatorConfig *controllerv1alpha1.OperatorConfiguration) error {
+func updateOperatorConfig(devEnvironments chev2.CheClusterDevEnvironments, operatorConfig *controllerv1alpha1.OperatorConfiguration) error {
 	var pvc *chev2.PVC
+	disabledContainerBuildCapabilities := utils.GetBool(devEnvironments.DisableContainerBuildCapabilities, true)
```

Comment:
Can we use [1] here?
https://github.com/eclipse-che/che-operator/blob/main/api/v2/checluster_types.go#L808